### PR TITLE
Add @studio/changes

### PR DESCRIPTION
**Pull request by mroderick**
This will help build a changelog and prevent missing tags in the shared repository

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,10 @@
     "test": "TZ=UTC mocha --reporter dot 'lib/**/*.test.js'",
     "test-check-coverage": "npm run test-coverage && nyc check-coverage --branches 100 --functions 100 --lines 100",
     "test-coverage": "nyc --all --reporter text --reporter html --reporter lcovonly npm run test",
-    "demo": "mocha demo/*.test.js"
+    "demo": "mocha demo/*.test.js",
+    "preversion": "npm test-check-coverage",
+    "version": "changes",
```
Comment (mantoni) [inline package.json]: No `--commits` and no `--footer`?

Comment (mroderick) [reply package.json]: I used `node_modules/.bin/changes --init` to generate the entries ...

**Comment by mroderick** [comment]
I pushed an extra commit, adding `--commits --footer`

**Comment by mroderick** [comment]
Once happy, I'll squash and merge

**Pull request merged**